cypress/plugins/index.js

```diff
@@ -46,6 +46,19 @@ module.exports = (on, config) => {
               },
             ],
           },
+          {
```

Comment:
I think Cypress supports TS by default now. You just need to add in the extensions like you have above and then create a new `tsconfig.json` file in the cypress dir like [here](https://github.com/bbc/simorgh/pull/10235/files#diff-7c73a5b98f30e40574b7149a83a19e335c5ee48b8c9c58e4e6cbfe6a07777037)

Reply:
I think the `tsconfig.json` file within the Cypress directory is used for any Cypress tests that we write in TS, rather than Cypress importing TS files from the `app` folder. I think we still need the `ts-loader` to transpile TS files to be used in Cypress.